automation/pages/landingPage.py

```diff
@@ -0,0 +1,27 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.basePage import Page
+
+class landingPage(Page):
+    #Elements
+
+    #Locators
+    loginButton_loc = (By.CLASS_NAME, "spotify-login")
+    
+    def __init__(self, driver):
+        Page.__init__(self, driver)
```

Comment:
Could be wrong but it kinda looks like you are just calling the prototype (maybe its a feature of Python I am unaware of...) 

I would expect `super(Page, self).__init__(driver)` to call the base class constructor